src/store/index.js

```diff
@@ -0,0 +1,11 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import rootReducer from './counterData/reducer';
+
+const composeEnhancers = composeWithDevTools({ trace: true, traceLimit: 25 });
+const reducers = combineReducers({
+  rootReducer,
```

Comment:
please create another reducers intuitively, it should contain only that things which are similar.